mobile-app/app/screens/AppNavigator/screens/Balances/components/Announcements.tsx

```diff
@@ -24,14 +26,58 @@ export function Announcements (): JSX.Element {
     hiddenAnnouncements,
     hideAnnouncement
   } = useDisplayAnnouncement()
+  const isBlockchainDown = useBlockchainStatus()
+  const deFiChainStatusUrl = 'https://status.defichain.com/'
 
+  const blockChainIsDownContent: AnnouncementData[] = [{
+    lang: {
+      en: 'We are currently investigating a syncing issue on the blockchain. View more details on the DeFiChain Status Page.',
+      de: 'Wir untersuchen derzeit ein Synchronisierungsproblem der Blockchain. Weitere Details auf der DeFiChain Statusseite.',
+      'zh-Hans': '我们目前正在调查区块链上的同步化问题。前往 DeFiChain Status 页面 了解更多状态详情。',
+      'zh-Hant': '我們目前正在調查區塊鏈上的同步化問題。前往 DeFiChain Status 頁面 了解更多狀態詳情。',
+      fr: 'Nous enquêtons actuellement sur un problème de synchronisation sur la blockchain. Voir plus de détails sur DeFiChain Status Page.'
+    },
+    version: '0.0.0',
+    url: {
+      ios: deFiChainStatusUrl,
+      android: deFiChainStatusUrl,
+      windows: deFiChainStatusUrl,
+      web: deFiChainStatusUrl,
+      macos: deFiChainStatusUrl
+    }
+  }]
+
+  const [emergencyMsgContent, setEmergencyMsgContent] = useState<AnnouncementData[] | undefined>()
   const announcement = findAnnouncementForVersion(nativeApplicationVersion ?? '0.0.0', language, announcements)
-  const displayAnnouncement = getDisplayAnnouncement(hiddenAnnouncements, announcement)
+  const emergencyAnnouncement = findAnnouncementForVersion('0.0.0', language, emergencyMsgContent)
+  const existingAnnouncements = getDisplayAnnouncement(hiddenAnnouncements, announcement)
+  const displayAnnouncement = emergencyAnnouncement !== null || existingAnnouncements
+  const announcementToDisplay = emergencyAnnouncement ?? announcement
 
-  if (!isSuccess || announcements?.length === 0 || announcement === undefined || !displayAnnouncement) {
+  useEffect(() => {
+    // To display warning message in Announcement banner when blockchain is down for > 45 mins
+    if (isBlockchainDown) {
+      setEmergencyMsgContent(blockChainIsDownContent)
+    } else {
+      setEmergencyMsgContent(undefined)
+    }
+  }, [isBlockchainDown])
+
+  if (!isSuccess || !displayAnnouncement || (announcementToDisplay == null) || ((emergencyAnnouncement == null) && !existingAnnouncements)) {
```

Comment:
I think we can simplify this even more.

The last condition `((emergencyAnnouncement == null) && !existingAnnouncements)` can be already handled by `displayAnnouncement` variable. You can update it to 

`const displayAnnouncement = emergencyAnnouncement !== null || existingAnnouncments !== null`

Reply:
the condition can't be modified as !existingAnnouncements is required to close any displayed announcement